app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
```

Comment:
I think this should be `Admin` rather than `Dashboard`.  This follows the existing pattern.